element-hq/element-x-android
# Hide images in the timeline

## Content


Add a switch to the developer settings screen to prevent preview of images, videos and sticker to be rendered automatically in the timeline.
When enabled:
- the blurhash (if available) will be rendered by default, or a color surface will be rendered.
- A button exist to render the image
- The image is not clickable, it must be rendered first
- The thumbnail (for replyied image) is not rendered, expect if the original image has been rendered.
- The image does not render in the notification

## Motivation and context


Closes #3386 

## Screenshots / GIFs



Waiting for the final design before recording the screenshot.

## Tests



- Navigate to the developer setting to enable the new option
- Go to a room and observe that the images are not rendered automatically

## Tested devices

- OS version(s):

## Checklist
